cachelib/cachebench/cache/Cache-inl.h

```diff
@@ -620,10 +620,19 @@ Stats Cache<Allocator>::getStats() const {
     aggregate += poolStats;
   }
 
+  std::map<PoolId, std::map<ClassId, ACStats>> allocationClassStats{};
+
+  for (size_t pid = 0; pid < pools_.size(); pid++) {
+    auto cids = cache_->getPoolStats(static_cast<PoolId>(pid)).getClassIds();
+    for (auto cid : cids)
+      allocationClassStats[pid][cid] = cache_->getACStats(pid, cid);
```

Comment:
I think what you want here is at 
cache_->getPoolStats(poolId).mpStats.acStats is unordered_map<ClassId, ACStats> for the pool.

So you don't have to add a new API into CacheAllocator.h (or Cache)

Reply:
Yes, you right, we don't need this extra API. Done.